### DISCUSSION TITLE
Knowledge transfer to Keith

### DISCUSSION THREAD

**Discussion by mrosseel** [WimDeMeester/eFinder#23] [General]
Hi, just a general question, with the current pace of changes in the codebase, how do we keep Keith in-sync? Would be a shame if he forked off. And would be great if he joined here :)

**Comment by WimDeMeester** [comment]
I will contact Keith later this week.  Up to now, I alway send the latest version of our code to Keith and he sends his updated code to us.

**Reply by mrosseel** [reply]
cool, yeah ask him if the poetry thing is working out for him, that's the biggest change he'll see I guess.

**Reply by WimDeMeester** [reply]
Mail sent to Keith, with invitation to join here.
